osu.Game/Overlays/Dashboard/Home/News/HomeNewsPanelFooter.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Online.API.Requests.Responses;
+
+namespace osu.Game.Overlays.Dashboard.Home.News
+{
+    public abstract class HomeNewsPanelFooter : CompositeDrawable
```

Comment:
I'm not sure I agree with the design of this. I'd actually slightly prefer if the two inheritors of this were separated.

The reasoning is that this control is too nebulous to be useful, and the differences in design don't help (having the misspelled `BarPading` to control how the separator is displayed feels all sorts of wrong).